include/PlayHandle.h

```diff
@@ -48,11 +48,13 @@ class LMMS_EXPORT PlayHandle : public ThreadableJob
 public:
 	enum class Type
 	{
-		NotePlayHandle = 0x01,
-		InstrumentPlayHandle = 0x02,
-		SamplePlayHandle = 0x04,
-		PresetPreviewHandle = 0x08
-	} ;
+		NotePlayHandle,
+		InstrumentPlayHandle,
+		SamplePlayHandle,
+		SamplePreviewPlayHandle,
+		PresetPreviewHandle
+	};
+
 	using Types = Flags<Type>;
```

Comment:
These look like they were intended to be bit-flags, and your changes will break that. If this is intentional that's probably fine (is it even valid for a handle to be more than one of these at a time?) but the `using Types = Flags<Type>` leads me to believe the rest of the code may still be expecting them to be bit-flags.

Reply:
Yeah, I didn't know why they were being set to specific values so I removed it. I'm not sure why the code would need to treat these as bit flags 🤔.

Reply:
I'd double-check anywhere `PlayHandle::Types` is used. If bit-flags aren't needed, `Types` should be removed. I can take a closer look later to help out.

Reply:
Finally got around to checking and it looks like it's used by `AudioEngine::removePlayHandlesOfTypes()`, specifically to quickly determine whether to remove a play handle or not through a simple bitmask operation. These should probably stay bit flags then.